dist/browser-detect.js

```diff
@@ -0,0 +1,138 @@
+define(['browser-detect'], function() {
```

Comment:
@lfender6445 what are you thoughts on adding few more helper methods, AG just had a case that we had to put a wrapper around one of the calls `.platform` https://github.com/rentpath/ag/commit/9ccb8c8380df30ccc0fed90d38fe457a6050c426
in ag-utils.js